inputFiles/poromechanics/PoroElastic_hybridHexPrism_co2_sequential_3d.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" ?>
+
+<Problem>
+
+  <Included>
+    <File name="./PoroElastic_hybridHexPrism_co2_base.xml" />
+  </Included>
+  
+  <Solvers>
+
+    <MultiphasePoromechanics
+      name="multiphasePoroelasticity"
+      solidSolverName="linearElasticity"
+      flowSolverName="twoPhaseFlow"
+      logLevel="1"
+      initialDt="1e11"
+      targetRegions="{ reservoir }">
+      <NonlinearSolverParameters
+        newtonTol="1.0e-8"
+	couplingType="Sequential"  
+        lineSearchAction="None"
```

Comment:
Does the current implementation handle lineSearch cuts properly?

Reply:
No, I don't think the current implementation can combine line search and sequential. So, after your comment I added a check to let the user know that it will not work:
https://github.com/GEOSX/GEOS/blob/471733282b52f510018a7520912c77bd0860c3e6/src/coreComponents/physicsSolvers/multiphysics/CoupledSolver.hpp#L536-L545